react1/readme.md

```diff
@@ -2,10 +2,8 @@
 
 ## In this 3 week module we will learn React!
 
-## Planning
-
-| Week | Topic                                                                                                                                                                                                                                                                                                                                                                                                         | Preparation                         | Lesson plan                         | Homework                      |
-| ---- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------- | ----------------------------------- | ----------------------------- |
-| 1.   | [Thinking the React way](https://reactjs.org/docs/thinking-in-react.html)<br> <li> What was the initial problem React solves?<br> Understanding the component model<br> <li> Seeing a website as components<br> <li> Thinking in abstractions<br> JSX transition into building components<br> Setting up the React App (Create-react-app) <br>[Props](https://reactjs.org/docs/components-and-props.html)<br> | [Preparation](week1/preparation.md) | [Lesson plan](week1/lesson-plan.md) | [Homework](week1/homework.md) |
-| 2.   | **Building components** Component tree <br> <li> From sketch of todolist to component tree <br> Stateful logic - Having logic around state <br> List keys <br> Life cycle <br> Lifting state                                                                                                                                                                                                                  | [Preparation](week2/preparation.md) | [Lesson plan](week2/lesson-plan.md) | [Homework](week2/homework.md) |
-| 3.   | **Advanced React** <br> Forms and events <br> Using data fetching <br> Children <br> Prop types                                                                                                                                                                                                                                                                                                               | [Preparation](week3/preparation.md) | [Lesson plan](week3/lesson-plan.md) | [Homework](week3/homework.md) |
+| Week | Topic | Preparation | Lesson plan | Homework |
```

Comment:
For consistency with the other modules, can the homework links go to the readme instead of directly to the repo?